addons/io_scene_gltf2/blender/exp/gltf2_blender_gather_animations.py

```diff
@@ -45,29 +48,37 @@ def gather_animations(blender_object: bpy.types.Object, export_settings) -> typi
         # Set action as active, to be able to bake if needed
         if blender_object.animation_data: # Not for shapekeys!
             if blender_object.animation_data.action is None \
-                    or (blender_object.animation_data.action.name != blender_action.name):
+                    or (blender_object.animation_data.action.name != blender_action[0].name):
```

Comment:
I think it would be better here to change the loop above to

    for blender_action, track_name in blender_actions

You wouldn't need to insert all the `[0]`s and the part below with `blender_action[1]` would be clearer too.